.github/CHANGELOG.md

```diff
@@ -485,9 +463,6 @@ random_mat2 = rng.standard_normal(3, requires_grad=False)
 *  A decomposition has been added for the `qml.CSWAP` operation.
```

Comment:
I can't comment on it, but above, on the entry:

> * Implement special handling for measuring the variance of Projector observables to improve memory usage.
> [(#1368)](https://github.com/PennyLaneAI/pennylane/pull/1368)

I would delete this, and simply add a link to #1368 to the existing projector entry,

> * Added Projector observable, which is available on all devices inheriting from